runtime/jvmti/jvmtiHelpers.c

```diff
@@ -670,7 +679,7 @@ getThreadState(J9VMThread *currentThread, j9object_t threadObject)
 			state |= JVMTI_THREAD_STATE_IN_NATIVE;
 		}
 #else /* J9VM_INTERP_ATOMIC_FREE_JNI */
-		if ( (vmThread->omrVMThread->vmState & J9VMSTATE_MAJOR) == J9VMSTATE_JNI ) {
+		if ((vmThread->omrVMThread->vmState & J9VMSTATE_MAJOR) == J9VMSTATE_JNI) {
```

Comment:
Did you want to put the constant on the left since this line is already modified?

Reply:
The change to this file is not required now, so skip format only change.